### DISCUSSION TITLE
提交了meta 的表格

### DISCUSSION THREAD

**Discussion by Alexzhang369** [ymcui/Chinese-LLaMA-Alpaca#568] [General]
![image](https://github.com/ymcui/Chinese-LLaMA-Alpaca/assets/13427946/eac80030-a5b3-4469-884a-af4755d18484)

感觉遥遥无期，得找找其他法子了。

**Comment by ymcui** [comment]
请不要频繁开issue讨论这个问题，谢谢。
关于原版权重，网上已经有很多替代方法了，善用搜索自己多找一下吧。
